docs/modules/editable-layers/api-reference/edit-modes.md

```diff
@@ -1,14 +1,16 @@
-# Editing Modes
+# Editor Modes
```

Comment:
(Optional) I think "Edit Modes" might feel more consistent here:

```suggestion
# Edit Modes
```

Reply:
It was trying to avoid of same title for group and title in the TOC. Split into multiple files to avoid the problem and make it easier to browse the modes.